app/directives/selectionList/selectionList.directive.js

```diff
@@ -37,7 +37,7 @@ angular.module('dashboard')
                 if (!(att instanceof AttachmentData)) {
                     $log.error("dbSelectionList: isDisabled: unsupported arg type: " + JSON.stringify(att));
                 } else {
-                    res = !angular.isString(att.link) || !att.link.length;
+                    res = (att.publicity === CONST.PUBLICITY.SECRET) || !angular.isString(att.link) || !att.link.length;
```

Comment:
Maybe add a comment that disabled for secret docs because currently no easy way to popup them?